src/docs/sphinx/dev_guide/profiling.rst

```diff
@@ -3,37 +3,79 @@
 .. ##
 .. ## SPDX-License-Identifier: (BSD-3-Clause)
 
-=============================
-Profiling Serac using Caliper
-=============================
+=======================================
+Profiling Serac using Adiak and Caliper
+=======================================
+
+Introduction to Adiak
+---------------------
+
+`Adiak <https://github.com/LLNL/Adiak>`_ is a library developed at LLNL for collecting
+metadata that can be used to compare multiple runs across programs.  The full documentation
+can be found `here <https://github.com/LLNL/Adiak/blob/master/docs/Adiak%20API.docx>`_,
```

Comment:
Unfortunately these need to be uniquely named and will cause a warning, seen here:

https://dev.azure.com/llnl-serac/serac/_build/results?buildId=6359&view=logs&j=1be3ff84-5b97-5584-c1a4-7ebbbeb3b802&t=d32b9d99-ea3e-5518-9381-52c611e33fea&l=2432

Reply:
Oh, I didn't know that about rst. I believe I fixed it.